### ISSUE TITLE
rabin2 fails to parse Windows PDB symbol file.

### ISSUE THREAD

**Issue by n00bSec** [radare/radare2#9300]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             |  Ubuntu-16.04.2 x86 64
| File format of the file you reverse (mandatory)      | PDB
| Architecture/bits of the file (mandatory)            | x86/64
| r2 -v full output, **not truncated** (mandatory)         |radare2 2.3.0 17272 @ linux-x86-64 git.2.2.0-361-g8b1e0bb commit: 8b1e0bb16e738d9e805297bca21adcac49a6b0a9 build: 2018-02-02__22:51:05
| r2 -V full output in a pastebin document (mandatory) | https://gist.github.com/n00bSec/e5e488935fa925a4553a622886f27065

### Expected behavior
Running `rabin2 -Pr xul.pdb` should export symbols out of the PDB file into a format that r2 should be able to use.

### Actual behavior
No symbols are exported, and instead a long string of "oops" messages are generated.

```
parse_sval()::oops
parse_sval()::oops
parse_sval()::oops
parse_sval()::oops
parse_sval()::oops
parse_sval()::oops
get_sval_name::oops
get_sval_val::oops
get_sval_name::oops
get_sval_name::oops
get_sval_val::oops
get_sval_name::oops
get_sval_name::oops
get_sval_val::oops
get_sval_name::oops
get_sval_name::oops
get_sval_val::oops
get_sval_name::oops
...
```

### Steps to reproduce the behavior 
1. Rebuild r2 from source on a Linux machine.
2. On a Windows system, download the symbols for Firefox 58 from Mozilla's symbol server using the documentation here: https://developer.mozilla.org/en-US/docs/Mozilla/Using_the_Mozilla_symbol_server
3. Move the xul.pdb file from the created symcache folder on the Windows to a Linux machine.
4. Run `rabin2 -Pr xul.pdb`.

Very slowly, the oops messages should be written to the console.
https://asciinema.org/a/awLVw9nfabTwLzXF5z6f3MN8l

The same issue may exist on a Windows machine, but I have only used r2 on Linux.

### Additional Logs, screenshots, source-code,  configuration dump, ...

**Comment by Maijin** [comment]
You can change the symbol server and make the download automatically too

**Comment by Maijin** [comment]
@xarkes may want to take a look at this issue

**Comment by riverar** [comment]
Can reproduce similar issue on Windows/x86 using https://symbols.mozilla.org/xul.pdb/9293832038AC49D18F68427E049C83172/xul.pd_, am investigating.

**Comment by riverar** [comment]
These messages get printed out due to missing support for various symbol types (e.g. QUADWORD, UQUADWORD, SHORT, etc.), but are not critical errors; just warnings.

The pdb is huge (620MB+), hence the slower than usual parse time.

I think I should have this fixed up today (and change those unhelpful warnings) if I don't cause secondary problems elsewhere. But the only impact I see here is that you won't find/import the above-referenced types. Everything else should work great.

**Comment by riverar** [comment]
@n00bSec can we close this issue? We cleaned up the output. Feel free to create another "xul.pdb takes hours to parse" issue for tracking that.

**Comment by Maijin** [comment]
@riverar next time use "blablab fix #9300" in your commit title so issue get closed automagically when merged ;)
